portfolio/src/main/webapp/background.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Hadley Shapland</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body>
+        <!-- SIDEBAR -->
+        <div class="sidenav">
+            <a href="index.html">Home</a>
+            <a href="background.html">Background</a>
+            <a href="experience.html">Experience</a>
+            <a href="funfacts.html">Fun Facts</a>
+        </div>
+        <!-- END SIDEBAR -->
+
+        <div id="content">
+            <h1>BACKGROUND</h1>
+            <p>My name is Hadley Shapland and I am a 
+            <a href="https://buildyourfuture.withgoogle.com/programs/step/">STEP Intern</a> 
+            at Google. I am a Computer Science and Math major at Vanderbilt University and will 
+            be starting my junior year this fall. I am from Champaign, IL and ...</p>
+           
+           <img src="images/kip.jpg"/>
```

Comment:
maybe add an alt tag to the image?

(see https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback)

Reply:
Good idea, fixed this in new commit.